lib/hanami/cli/generators/gem/app/app_layout.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title><%= humanized_app_name %></title>
+    <%- if bundled_assets? -%>
+    <%%= favicon %>
+    <%- end -%>
+  </head>
+  <body>
+    <%%= yield %>
```

Comment:
Nice pickup with needing `<%%=` here. Thank you! 🙇🏼

Reply:
Note for the future: because this is an ERB template that needs to output a `.erb` file if we don't use the escape `<%%=`, we can't output `<%= yield %>`. The ERB template of the code generators will attempt to yield immediately.